docs/deployment_patterns/how-to-instantiate-a-data-context-on-databricks-spark-cluster.md

```diff
@@ -5,11 +5,11 @@ import Prerequisites from '../guides/connecting_to_your_data/components/prerequi
 
 This guide will help you instantiate a Data Context on an Databricks Spark cluster.
 
-The guide demonstrates the recommended path for instantiating a Data Context without a full configuration directory and without using the Great Expectations [command line interface (CLI)](../guides/setup/configuring-data-contexts/how-to-create-a-new-data-context-with-the-cli.md).
+The guide demonstrates the recommended path for instantiating a Data Context without a full configuration directory and without using the Great Expectations [command line interface (CLI)](../guides/setup/configuring_data_contexts/how-to-create-a-new-data-context-with-the-cli.md).
```

Comment:
@cdkini GitHub still shows me file names containing hyphens, while the rest of the path uses the underscores.  Thanks.